app/config/config.yml

```diff
@@ -97,8 +97,17 @@ api_platform:
   title: PrestaShop API
   version: 1.0.0
   enable_entrypoint: false
+  formats:
+    json:
+      mime_types: [ 'application/json' ]
+    jsonld:
```

Comment:
Does it mean json is the default format when no one is specified in the request header? Then why did you need to add the header in all the auto tests?

Also, did you keep the jsonld format to be retro-compatible with the existing UI tests?
It may be risky because it kind of imply we do handle this format even if we don't really test it. Having this format was not part of the initial MVP so I think it would be less risky not to handle it at all at least for now

Reply:
Default header are application/json+ld on request method in tests. I have to keep jsonld because for an obscur reason, other tests that does not concern api was failed. But add json in first is the way to make it default parameter.